testing/percona-xtradb-cluster/APKBUILD

```diff
@@ -0,0 +1,226 @@
+url="https://www.percona.com/software/mysql-database/percona-xtradb-cluster"
+pkgusers="mysql"
+pkggroups="mysql"
+arch="all"
+license="GPL2"
+depends="$pkgname-common $pkgname-server $pkgname-client"
+depends_dev="openssl-dev zlib-dev"
+makedepends="cmake openssl-dev zlib-dev readline-dev libaio-dev ncurses-dev linux-headers bison bsd-compat-headers"
+install="$pkgname.pre-install"
+source="https://github.com/percona/percona-xtradb-cluster/archive/$_pkgname-$_pkgver.tar.gz
+        fix-posix_timers.patch
+        my.cnf
+        percona-xtradb-cluster.cnf
+        mysqld.cnf
+        mysqld_safe.cnf
+        client.cnf
+        wsrep.cnf
+        "
+
+subpackages="$pkgname-doc $pkgname-dev $pkgname-common $pkgname-client-libs:_client_libs
+        $pkgname-client $pkgname-server $pkgname-test:mytest"
+
+builddir="$srcdir/$pkgname-$_pkgname-$_pkgver"
+
+build() {
+        cd "$builddir"
+
+        source "VERSION"
+        local MYSQL_VERSION="$MYSQL_VERSION_MAJOR.$MYSQL_VERSION_MINOR.$MYSQL_VERSION_PATCH"
+        local PERCONA_SERVER_EXTENSION="$(echo $MYSQL_VERSION_EXTRA | sed 's/^-/rel/')"
+
+        local WSREP_VERSION="$(grep WSREP_INTERFACE_VERSION wsrep/wsrep_api.h | cut -d '"' -f2).$(grep 'SET(WSREP_PATCH_VERSION' "cmake/wsrep.cmake" | cut -d '"' -f2)"
+
+        local COMMENT="Percona XtraDB Cluster binary (GPL) $MYSQL_VERSION-$WSREP_VERSION"
+
+        CFLAGS="$CFLAGS -DPERCONA_INNODB_VERSION=$PERCONA_SERVER_EXTENSION -DSIGEV_THREAD_ID=4 -U_FORTIFY_SOURCE"
+        CXXFLAGS="$CXXFLAGS -DPERCONA_INNODB_VERSION=$PERCONA_SERVER_EXTENSION -U_FORTIFY_SOURCE"
+
+        cmake . -DBUILD_CONFIG=mysql_release \
+                -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+                -DSYSCONFDIR=/etc/mysql \
+                -DWITH_EMBEDDED_SERVER=OFF \
+                -DFEATURE_SET=community \
+                -DENABLE_DTRACE=OFF \
+                -DWITH_SSL=system \
+                -DWITH_ZLIB=system \
+                -DCMAKE_INSTALL_PREFIX=/usr \
+                -DMYSQL_DATADIR=/var/lib/mysql \
+                -DINSTALL_INFODIR=share/mysql/docs \
+                -DINSTALL_MANDIR=share/man \
+                -DINSTALL_PLUGINDIR=lib/mysql/plugin \
+                -DINSTALL_SCRIPTDIR=bin \
+                -DINSTALL_INCLUDEDIR=include/mysql \
+                -DINSTALL_DOCREADMEDIR=share/mysql \
+                -DINSTALL_SUPPORTFILESDIR=share/mysql \
+                -DINSTALL_MYSQLSHAREDIR=share/mysql \
+                -DINSTALL_DOCDIR=share/mysql/docs \
+                -DINSTALL_SHAREDIR=share/mysql \
+                -DMYSQL_UNIX_ADDR=/run/mysqld/mysqld.sock \
+                -DDEFAULT_CHARSET=utf8 \
+                -DDEFAULT_COLLATION=utf8_general_ci \
+                -DMYSQL_SERVER_SUFFIX="-$WSREP_VERSION" \
+                -DWITH_INNODB_DISALLOW_WRITES=ON \
+                -DDOWNLOAD_BOOST=1 \
```

Comment:
Alpine policy is to use system provided libraries; not to bundle them. There are few exceptions, but we'd rather not add more things.